xc/device.ml

```diff
@@ -1511,7 +1524,10 @@ module Vusb = struct
            Also need to do deprivileged work in usb_reset script if QEMU is deprivileged.
         *)
         begin match Qemu.pid ~xs domid with
-          | Some pid -> usb_reset_attach ~hostbus ~hostport ~domid ~pid ~privileged
+          | Some pid ->
+            let stdout = usb_reset_attach ~hostbus ~hostport ~domid ~pid ~privileged in
+            let uid_gid = String.trim stdout |> Stdext.Xstringext.String.split ' ' in
```

Comment:
What happens where if the script fails? Can we intercept the error and respond with a reasonable exception?
Also, don't use `Stdext.Xstringext.String.split ' '` and do instead `String.split_at_char ' '`

Reply:
Thanks, @mseri. If the script failed, we will catch the error from ` exec_usb_reset_script`.

So `Stdext.Xstringext.String.split` is not used anymore?